c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -798,47 +798,48 @@ cudaError_t cudaMallocMaybeCapturing(void** p, size_t size) {
   }
 }
 
-class TraceEntryRingBuffer {
+template <class T>
+class CCARingBuffer {
  public:
-  TraceEntryRingBuffer() {
+  CCARingBuffer() {
```

Comment:
nit: can just be RingBuffer, it is local to this file.

Reply:
Good point! Changed